client/state/utils/README.md

```diff
@@ -0,0 +1,18 @@
+Utils
+========
+
+A module for those little helpers that need to interact woth the store.
+
+# Actions
+Used in combination with the Redux store instance `dispatch` function, actions can be used in manipulating the current global state.
+
+## uniqueId( prefix )
+Meant as a replacement for _ and lodash "uniqueId" function.
+Returns unique numeric prefixed by prefix and increments internal state, so the next uniqueId will be different.
```

Comment:
> I let myself translate this source to ES5, and as I understand it, it gets returned:

Oh yeah, d'oh, you're absolutely right. I'd read it too fast. :)

> I took this approach to have an 'atomic' action that both increments value and returns it.
The regular redux approach would be to have action for incrementing and selector for accessing, so to get the next id, the flow would be: […]

Hm-hm, there's a lot of indirection. It's a tricky issue.

Don't get me wrong, the current implementation works and is very straightforward, and that is good. I'm being cautious because we're diverging from our standard way of thinking about global data, and I'm also wondering whether we should avoid diverging altogether **or** find a standard way to diverge. :)

In a way, the issue at hand is that we want some sort of non-component, non-UI state in our app to be Redux- (and thus, for instance, SSR-) friendly. Right? If we didn't have those concerns, a simple module-level global `_count` variable — or a closure-bound variable — would have done the job. Function memoization is also something that would fall in this category: its cache is a form of persistent/global data, and in an SSR context we'd want that data bound to each request _or_ to each React render tree (i.e. we wouldn't want it globally shared across requests).

So, where I'm getting at is that we might want a sort of standard way to bind some stateful function (`uniqueId`, `memoize`) to a React tree/context/request. Though that likely won't be solved in just this PR, it's something to keep thinking about.